pace-util/HISTORY.md

```diff
@@ -17,6 +17,8 @@ Minor changes:
 - added `CubedSphereCommunicator.from_layout` constructor method
 - added support for built-in `datetime` in ZarrMonitor
 - `edge_interior_ratio` is now an optional argument of `tile_extent_from_rank_metadata`
+- added `report` for saving performance timing outputs
+- added `Barrier` in `NullComm` which behaves the same as `barrier`
```

Comment:
NullComm is in `fv3core` and report is in `driver`, so you don't need these in the pace-util history.

We could move NullComm to `pace-util` later but report definitely seems to be long in `driver`.

```suggestion
```